Gemfile

```diff
@@ -22,6 +22,8 @@ gem 'importmap-rails'
 gem 'turbo-rails'
 
 # Hotwire's modest JavaScript framework [https://stimulus.hotwired.dev]
+gem 'ffi', '~> 1.15', '>= 1.15.5'
+gem 'rails-controller-testing'
 gem 'rspec-rails', '~> 4.0.0'
```

Comment:
- [OPTIONAL] Please let's add this gem to the development group to expose generators and rake tasks without having to type RAILS_ENV=test. So let's do like this:

```ruby
group :development, :test do
 ...
gem 'rspec-rails', '~> 4.0.0'
end
```

Reply:
Thanks